discordapi/const.py

```diff
@@ -27,7 +27,7 @@
     path = os.path.join(path, data)
     data = open(path).read()
     commit = data[:7]
-except (FileNotFoundError, IndexError):
+except (FileNotFoundError, IndexError, Exception):
```

Comment:
Using a general Exception object in except statement is not advised. Please properly report the issue and provide a specific solution.

Reply:
Actually, on a second thought, this could be the right way of handling this...
I'll merge this for now.